utils/db-parts.stanza

```diff
@@ -223,7 +223,7 @@ defn resistor-query-properties (user-properties:Tuple<KeyValue>,
                    sort,
                    operating-temperature,
                    "resistor",
-                   false)
```

Comment:
Let's review that together. I need see if I can share the real time Octopart quota with you so that you can take responsibility of not exhausting it.

Reply:
Also I wanted to implement per user quotas and potentially we could make users pay for using a lot. But I have not had time to do so.
Tracking their usage is a minimum if we want to enable it by default

Reply:
Otherwise you can exhaust the quota with a for loop. You already can but I know nobody reads the part db doc so nobody knows they can

Reply:
Sure, I would like to know also the licensing system in order to perform per machine or per user tracking.

Reply:
We have a Statistics database, we are already recording every call to the part db but we just know the person queried the part db, we need to have a new table in db with the octopart quotas per user. Octopart counts quota as number of parts returned so it is trackable

Reply:
The cofactr changes is merged to `jit-backend` in the testing branch. This PR can be merged once it is deployed to production.